02_calculator/calculator.rb

```diff
@@ -1 +1,17 @@
 #write your code here
+def add(a, b)
+    c = a + b
+end
+
+def subtract(a, b)
+    c = a - b
+end
+
+def sum(a)
+    c = 0
+    a.each  { |a|
+    c += a
+    }
+return c
+end
```

Comment:
If sum is an integer, or any other datatype other than an Array, it will throw an error. But other then that, the function works.

Reply:
It works better for me if you explain how to do it correctly, than if you explain how to produce eerros. Can you explain the solutions instead of errors? I want to learn how to fix something not how to make it wrong. If you explain errors to me, I will be learning to produce errors, I don't want to learn how to produce errors.